src/applications/my-education-benefits/config/form.js

```diff
@@ -780,12 +788,98 @@ const formConfig = {
                       }
                     },
                   ],
+                  'ui:options': {
+                    replaceSchema: formData => {
+                      const { livesOnMilitaryBase } = formData[
+                        'view:mailingAddress'
+                      ];
+                      const { country } = formData[
+                        'view:mailingAddress'
+                      ].address;
+
+                      if (
+                        // formData.showMEBMailingAddressForeign &&
+                        livesOnMilitaryBase &&
+                        country !== 'USA'
+                      ) {
+                        return {
+                          type: 'string',
+                          title: 'APO/FPO',
+                          enum: ['APO', 'FPO'],
+                          enumNames: ['APO', 'FPO'],
```

Comment:
Not sure if there was a requirements change, but is DPO supposed to be included here?